snapshotserver/main.go

```diff
@@ -153,6 +156,22 @@ func Close() {
 	}
 }
 
+func basicValidationHandler(h httprouter.Handle) httprouter.Handle {
+	var re = regexp.MustCompile(scopeParamValidChars)
```

Comment:
The scope validation check is moving to **snapshot.go**, but we can make it a global var over there for this purpose.